lib/elixir/lib/calendar/naive_datetime.ex

```diff
@@ -194,8 +194,10 @@ defmodule NaiveDateTime do
       {:error, :invalid_time}
       iex> NaiveDateTime.new(2000, 1, 1, 23, 59, 60, 999_999)
       {:error, :invalid_time}
+
+      # Invalid microsecond (out of the valid range)
       iex> NaiveDateTime.new(2000, 1, 1, 23, 59, 59, 1_000_000)
-      {:error, :invalid_time}
+      ** (FunctionClauseError) no function clause matching in NaiveDateTime.new/8
```

Comment:
Please revert for the reasons said in `time.ex`.

Reply:
done